hoot-core/src/main/cpp/hoot/core/visitors/ReportMissingElementsVisitor.cpp

```diff
@@ -57,17 +57,17 @@ void ReportMissingElementsVisitor::_reportMissing(ElementId referer, ElementId m
   {
     if (_removeMissing)
     {
-      LOG_WARN("Removing missing " << missing.toString() << " in " << referer.toString() << ".");
+      LOG_INFO("Removing missing " << missing.toString() << " in " << referer.toString() << ".");
```

Comment:
Isn't it appropriate to have this as a warning? We can silence the warnings in the test in another way if you want.

Reply:
Check out this class for temporarily disabling log messages: https://github.com/ngageoint/hootenanny/blob/master/hoot-core/src/main/cpp/hoot/core/util/Log.h#L146

Reply:
I changed this because there are tests that spit out that warning and we don't get the whole set of unbroken "dots" like we ought.  For example from the output you sent me earlier:
```
...........................................................................15:06:45.272 WARN  ...portMissingElementsVisitor.cpp(  60) Removing missing Node:-252 in Relation:-157.
............15:06:46.017 WARN  ...portMissingElementsVisitor.cpp(  60) Removing missing Node:-596 in Relation:-313.
...............15:06:47.203 WARN  ...portMissingElementsVisitor.cpp(  60) Removing missing Node:-239 in Relation:-122.
15:06:47.203 WARN  ...portMissingElementsVisitor.cpp(  60) Removing missing Node:-247 in Relation:-121.
............................................................................................................................................................................................................................................................................
```
In my mind, this isn't warning worthy.  It is information that there is a node not in your data but is referenced in a relation and it is simply being removed.  Sounds more informational than warning-al.  If that is a warning level thing I can go back and change it and use the `DisableLog` class for those tests.

Reply:
I can see the rationale for an INFO message. Works for me.